proposals/(draft) 2023-02-upgrade-v2.0.6/README.md

```diff
@@ -0,0 +1,19 @@
+## Background
+This is a draft of our SW Upgrade proposal to do a software upgrade to v2.0.6 software tag of the Panacea codebase. The purpose of sharing this draft is to gather validators feedback before we submit the proposal.
+
+## Proposal
+Panacea team would like to propose that we upgrade our chain’s ibc-go from v2 to v4.3.0 (with cosmos-sdk v0.45.12 and informalsystems/tendermint v0.34.24) and remove CosmWasm from the chain.
+
+We believe that removing CosmWasm requires some explanation from our end whereas upgrading ibc-go is straightforward to most of you.
+Pancaea team first adopted CosmWasm to accomodate a possible NFT usage on the chain. However, because of below three reasons, we now think that it is more reasonable for us to remove CosmWasm from the chain.
```

Comment:
What if we also added the following?
```
More expensive to maintain due to many associated libraries compared to unused Wasm.
If we don't gain anything from Wasm, consider removing it.
```

Reply:
But... it looks like yours is similar to the one described here, so just a note.

Reply:
Thanks for the comment! I think we can leave it as it is becasue one of the three resaons say that there are no use cases for Wasm at this point. 

Thanks again!